components/filter/test/filter.visual-diff.js

```diff
@@ -226,7 +233,12 @@ describe('d2l-filter', () => {
 				await page.$eval(selector, async(filter) => {
 					await new Promise((resolve) => {
 						filter.addEventListener('d2l-localize-resources-change', resolve, { once: true });
-						document.querySelector('html').setAttribute('lang', 'ar');
+						if (document.querySelector('html').getAttribute('lang') !== 'ar') {
```

Comment:
> > Arabic in order to activate rtl
> 
> Right - it's possible the RTL test itself wants to use Arabic text, hadn't thought of that. Anyway I was just investigating flake, but I'm guessing if you just re-run the visual diff it'll pass.

That's true, I decided to keep the test in Arabic because they may have intended for it to test that. The reason the promise was not resolving was because it was waiting for the language change event to resolve on retries but the language had already changed so it times out. I added a check so that if the language had already changed it resolves the Promise